src/AnalogOutputFirmata.cpp

```diff
@@ -20,6 +20,11 @@
 #include "AnalogFirmata.h"
 #include "AnalogOutputFirmata.h"
 
+// Hack to compile for ESP32 boards until arduino-esp32 supports analogWrite
+#ifdef ESP32
+void analogWrite(byte pin, int value);
```

Comment:
Necessary because Arduino compiles all cpp files whether or not they are included.